lib/default.nix

```diff
@@ -232,6 +232,16 @@ in {
     inherit stdenv lib haskellLib srcOnly;
   };
 
+  # Do coverage of a package
+  coverageReport' = import ./cover.nix {
+    inherit lib haskellLib pkgs;
+  };
+
+  # Do coverage of a project
+  projectCoverageReport' = import ./cover-project.nix {
```

Comment:
Why the ticks? there aren't unticked versions here that I can see?

Reply:
There are the ones which are attributes on projects, but I think that's clear enough. We use the ticks when they're actually attributes of the same attribute set.

Reply:
Fixed: https://github.com/input-output-hk/haskell.nix/pull/762/files#diff-a013e20924130857c649dd17226282ffR235